libertree-frontend-ramaze/view/docs/en_GB/custom-css-file.markdown

```diff
@@ -0,0 +1,54 @@
+# Custom-CSS-File
+You can copy paste the code below in to your settings. Please put behind the semicolons a number in hexadecimal for the color. 
```

Comment:
I haven't tried the stylesheet with any colours yet.  What colour you choose is up to you.  Maybe it would be helpful to mark colours with a comment to remind the user to replace it.